src/views/statutory/PositionsList.vue

```diff
@@ -221,6 +225,10 @@ export default {
     },
     requirementsIsSet () {
       return (this.selectedPosition.requirements !== null && this.selectedPosition.requirements !== '')
+    },
+    // TODO: This is a temporary solution, in the future, we want to display either the "first" deadline OR the force_end deadline
```

Comment:
When we fix this, we should also think about what is good UI/UX wise. And that the final deadline might be earlier if enough people applied